#19 reinier eindopdracht

reinier303 opened the pull request:
#19
- Added player and camera movement that fit the gameplay.
- Added GameManager with score and reference to player.

reinier303 committed:
Added Project

reinier303 committed:
Revert "Added Project"

This reverts commit e79f4920b2ab373eea39d52391bd4408a870bf5a.

reinier303 committed:
Create .gitignore.txt

reinier303 committed:
Revert "Revert "Added Project""

This reverts commit 130074468d8f1b0c83fe9beb82e6a83744d263d6.

reinier303 committed:
Removed Project

reinier303 committed:
Delete .gitignore.txt

reinier303 committed:
Create .gitignore.txt

reinier303 committed:
Added Gitignore

reinier303 committed:
Add project and update gitignore

reinier303 committed:
Update .gitignore

reinier303 committed:
Updated Gitignore

reinier303 committed:
Updated Gitignore

reinier303 committed:
Cinemachine, Player and Bugfixes

Added Cinemachine, created a player and fixed a bug that would not make blocks kinematic

reinier303 committed:
GameManager and PlayerMovement

Created a GameManager and started on playermovement

reinier303 committed:
PlayerMovement

Player can now move

reinier303 committed:
Continuing player & camera movement

reinier303 committed:
Fixed some lowercase letters on lines

aaronvark commented:
Zorg ervoor dat je de issue reference in de description zet (in de titel pakt hij hem niet op).

aaronvark commented:
Het conflict hier kan worden geresolved door de edits aan die file compleet te deleten (dus alleen de master variant overhouden, kan gewoon in de web editor). Er is al een eigen ignore file namelijk in de project folder.

reinier303 committed:
Merge branch 'master' into #19-Reinier-Eindopdracht

reinier303 committed:
Added some comments for variables and changed a few variables

reinier303 committed:
Merge branch '#19-Reinier-Eindopdracht' of https://github.com/aaronvark/PeerReview1819 into Reinier-Eindopdracht

reinier303 committed:
Fixed wall sticking

Fixed the issue where the player stuck to walls by changing values in the character controller

BeTheBase reviewed (approved):
Lekker hoor, je moet alleen nog wel ff je object pooler gaan beveiligen met een interface en ik raad je aan ook wat te spelen met delegates en events.
